students/BrandonHenson/Lesson3/L3_Slicing.py

```diff
@@ -0,0 +1,46 @@
+# Brandon Henson
+# Lesson 3
+# Slicing
+# 4/4/18
+
+
+# with the first and last items exchanged
+def exchange_first_last(seq):
+    return(seq[-1:] + seq[1: -1] + seq[:1])
```

Comment:
The parentheses are not necessary here for the return statement. Same for all functions below.